src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptionhandlers/ErrorNotificationExceptionHandler.java

```diff
@@ -34,9 +40,12 @@ public ErrorNotificationExceptionHandler(MessageAutoCompletor autoCompletor) {
         ErrorNotificationException exception
     ) {
         if (exception.getStatus().is5xxServerError()) {
+            log.warn("Received server error from notification client", exception);
```

Comment:
Worth including message Id in those logs, so that we can track what happened to particular messages.